docs/reference/indices/analyze.asciidoc

```diff
@@ -51,6 +51,22 @@ curl -XGET 'localhost:9200/_analyze' -d '
 
 deprecated[5.0.0, Use `filter`/`token_filter`/`char_filter` instead of `filters`/`token_filters`/`char_filters`]
 
+If you want to try custom char filters/tokenizer/token filters,
+you can specify the settings of char filters/tokenizer/token filters in request body.
+
+[NOTE]
+This feature is only support in request body. You can't use custom settings in request parameters.
+
```

Comment:
I'd change the above two paragraphs as follows:

    Custom tokenizers, token filters, and character filters can be specified in the request body as follows: